lib/msal-browser/src/broker/nativeBroker/PlatformAuthExtensionHandler.ts

```diff
@@ -4,30 +4,31 @@
  */
 
 import {
-    NativeConstants,
+    PlatformAuthConstants,
     NativeExtensionMethod,
 } from "../../utils/BrowserConstants.js";
 import {
     Logger,
     AuthError,
     createAuthError,
     AuthErrorCodes,
-    AuthenticationScheme,
     InProgressPerformanceEvent,
     PerformanceEvents,
     IPerformanceClient,
 } from "@azure/msal-common/browser";
 import {
     NativeExtensionRequest,
     NativeExtensionRequestBody,
-} from "./NativeRequest.js";
+    PlatformBrokerRequest,
+} from "./PlatformBrokerRequest.js";
 import { createNativeAuthError } from "../../error/NativeAuthError.js";
 import {
     createBrowserAuthError,
     BrowserAuthErrorCodes,
 } from "../../error/BrowserAuthError.js";
-import { BrowserConfiguration } from "../../config/Configuration.js";
 import { createNewGuid } from "../../crypto/BrowserCrypto.js";
+import { PlatformBrokerResponse } from "./PlatformBrokerResponse.js";
```

Comment:
nit: We probably need to name it `PlatformAuthRequest` and  etc ..

Reply:
Let me follow this up in the next PR! Thanks Sameera!